osf/models/analytics.py

```diff
@@ -126,7 +126,12 @@ def update_counter(cls, page, node_info):
             # After we're sure this is stable, we can stop writing to the _id field, and query on
             # resource/file/action/version
             resource, file, action, version = cls.deconstruct_id(cleaned_page)
-            model_instance, created = cls.objects.select_for_update().get_or_create(_id=cleaned_page, resource=resource, file=file, action=action, version=version)
+            model_instance, created = PageCounter.objects.select_for_update().get_or_create(_id=cleaned_page)
```

Comment:
This is only temporary.  After `resource`, `file,``action`, and `version` columns have all been populated, and this is stable, we won't need to write or read the `_id` field anymore, we can fetch on the combination of the `resource`, `file`, `action`, and `version` fields. https://github.com/pattisdr/osf.io/pull/19/files